moped-api/claims.py

```diff
@@ -186,18 +186,23 @@ def format_claims(user_id: str, roles: list) -> dict:
     }
 
 
-def put_claims(user_email: str, user_claims: dict):
+def put_claims(user_email: str, user_claims: dict, cognito_uuid: str = None):
```

Comment:
This is nice, Sergio. I wondered if the data structure in the Cognito user payload would present a problem in the future when trying to get user attributes, and this is a great solution since we are already storing things in DynamoDB. 🙌